components/esp_modem/src/esp_modem_netif.cpp

```diff
@@ -99,6 +99,19 @@ void Netif::stop()
     signal.clear(PPP_STARTED);
 }
 
+void Netif::pause(bool do_pause)
```

Comment:
While this is a simple API it could be split in 2. And we have a return path with a boolean but the function signature states it return void. 

I would suggest split it in `Netif::start`, `Netif::pause` and maybe provide a `Netif::stop`

Reply:
> I would suggest split it in Netif::start, Netif::pause and maybe provide a Netif::stop

we already have start/stop.

> And we have a return path with a boolean

we don't. it was a return path from the lambda.

I have split the function into `pause()` and `unpause()` to make it clear. PTAL again.